Documentatie/EP-online-validaties.md

```diff
@@ -54,6 +55,7 @@ Onderstaande validaties worden allemaal en in willekeurige volgorde uitgevoerd.
 |  CheckContractor  															|	De contractor (degene die registreert) moet bestaan als gebruiker in EP-online, de rol contracthouder (CH) hebben en gemachtigd zijn voor registraties met de rekenmethodiek (TypeCalculation) in EP-online.
 |  CheckDuplicateAddresses  													|	Alle adressen (ZipCode+Number+Extension+BuildingAnnotation uit TPGIdentification) mogen slechts één keer voorkomen in het bestand.
 |  CheckDuplicateBagResidenceIds  												|	Alle verblijfsobjecten Id's (BagResidenceId) mogen slecht één keer voorkomen in het bestand.
+|  CheckEpcClassIdResidential													|	Bij een registratie voor woningbouw is het maximale labelletter A++++. 
```

Comment:
"maximale": ik weet niet of dit helemaal de beste term hiervoor is, maar ik weet momenteel niet helemaal wat we het beter kunnen noemen...